templates/TestStage.yml

```diff
@@ -87,7 +87,7 @@ stages:
 
       - task: DotNetCoreCLI@2
         displayName: Run Tests
-        timeoutInMinutes: 20
+        timeoutInMinutes: 25
         inputs:
```

Comment:
Does this still need increasing with the new xunit version? NetFx should be the worst case as we disable parallelism and in my PR this completed in 12 mins 31 seconds